discovery-infra/test_infra/cluster.py

```diff
@@ -0,0 +1,237 @@
+import logging
+
+from tests.conftest import env_variables
+from test_infra import consts, utils
+
+
+class Cluster:
+    
+    def __init__(self, api_client, cluster_name):
+        res = self._create(api_client, cluster_name)
```

Comment:
maybe lets allow to provide cluster id and if it is provided don't create one? just run get?

Reply:
I don't see the use case for this, if the way to create a cluster is by creating a Cluster obeject. wdty?